docs/docusaurus/versioned_docs/version-0.18/cloud/deploy_gx_agent.md

```diff
@@ -1,6 +1,6 @@
 ---
-sidebar_label: 'Deploy the GX Agent'
-title: 'Deploy the GX Agent'
+sidebar_label: '(Optional) Deploy the GX Agent'
+title: '(Optional) Deploy the GX Agent'
```

Comment:
The title will automatically be used as the sidebar label, so as long as they are supposed to be the same you can leave out the entry for `sidebar_label: '(Optional) Deploy the GX Agent'.

For conventions sake I would also remove "(Optional)" from the title of this page.  Within pages we typically only use "Optional" for prerequisites and some procedural steps, and when we do we follow the format: "Optional. Some thing that may or may not be needed." rather than using parenthesis.

When a guide itself is optional, we typically include that in the introduction material at the top of the guide rather than as part of the title (after all, the ToC isn't a procedure and almost every step has some element that is "optional" depending on what the user is trying to do!). 

I would update the opening paragraph of this guide with something like:
"""
GX Cloud is a fully hosted data quality solution and you do not need to deploy the GX Agent to make use of its full suite of features.  However, you can optionally deploy the GX Agent to exert additional control over how users access GX Cloud and how data is accessed by GX Cloud processes.

The GX Agent is an intermediary between GX Cloud and your organization's data stores. GX Cloud does not connect directly to your data with this deployment option, and all data access occurs within the GX Agent. GX Cloud sends jobs to the GX Agent, the GX Agent executes these jobs against your data, and then sends the job results to GX Cloud.

A local deployment of the GX Agent will allow you to test GX Cloud setup or processes from a single machine before moving to a shared production deployment.  Alternatively, you can deploy the GX Agent to your development environment and leverage GX Cloud while connecting to Data Sources using your organization's cloud infrastructure, for enhanced control and security.
"""

Reply:
Love this update!